tests/regress/sql/test_rejectmap_limit.sql

```diff
@@ -44,7 +40,7 @@ INSERT INTO s5.a SELECT generate_series(1,100000);
 
 SELECT diskquota.wait_for_worker_new_epoch();
 
-SELECT count(*) FROM master_log WHERE line LIKE '%the number of local quota reject map entries reached the limit%' AND line NOT LIKE '%LOG%';
+SELECT count(*) FROM gp_toolkit.__gp_log_master_ext WHERE logmessage LIKE '%the number of local quota reject map entries reached the limit%' AND logmessage NOT LIKE '%LOG%';
```

Comment:
```suggestion
SELECT count(*) FROM gp_toolkit.__gp_log_master_ext JOIN pg_stat_activity ON logsession = 'con'||sess_id WHERE pid = pg_backend_pid() AND logmessage LIKE '%the number of local quota reject map entries reached the limit%' AND logmessage NOT LIKE '%LOG%';
```

Reply:
Doesn't work, the session printing the warning here is somehow different from the current session. Still, the behavior should be identical to existing one (which simply reads the last line in the log file)